lib/Translation_unit.mli

```diff
@@ -15,6 +15,7 @@ type 'a t =
   { init_cmts: debug:bool -> Source.t -> 'a -> Cmt.t list -> Cmts.t
   ; fmt: debug:bool -> Source.t -> Cmts.t -> Conf.t -> 'a -> Fmt.t
   ; parse: Lexing.lexbuf -> 'a
+  ; make_parsable: string -> string
```

Comment:
I agree on renaming the function, but I don't see what having an option type here will achieve, it will force us to test both option values even though we know the appropriate recover function is always provided

Reply:
That was an API design comment. `string -> string` is great and it stay that way. Let me explain what I meant:

Suppose you're a user of that API and you want to create a value of type `'a t`. You have to provide a `make_parsable` function of type `string -> string` for your language right? Suppose you do not care about that and want to not use that feature. What should you pass? `fun x -> x`? `fun _ -> ""`? `fun _ -> assert false`? Difficult to tell without looking at the other uses! Asking for a `(string -> string) option` makes that a bit more obvious - you can pass `None`. It clarifies the fact that there's a natural default value. Of course, you need to add a `Option.value ~default:Fn.id` where you use that function. (it's also possible to keep storing `string -> string` but do this default value thing in a constructor)